src/coreComponents/linearAlgebra/interfaces/hypre/mgrStrategies/MultiphasePoromechanicsReservoirFVM.hpp

```diff
@@ -83,34 +83,34 @@ class MultiphasePoromechanicsReservoirFVM : public MGRStrategyBase< 4 >
     setupLabels();
 
     // Level 0
-    m_levelFRelaxType[0]         = MGRFRelaxationType::amgVCycle;
-    m_levelFRelaxIters[0]        = 1;
-    m_levelInterpType[0]         = MGRInterpolationType::jacobi;
-    m_levelRestrictType[0]       = MGRRestrictionType::injection;
-    m_levelCoarseGridMethod[0]   = MGRCoarseGridMethod::nonGalerkin;
-    m_levelGlobalSmootherType[0] = MGRGlobalSmootherType::none;
+    m_levelFRelaxType[0]          = MGRFRelaxationType::amgVCycle;
+    m_levelFRelaxIters[0]         = 1;
+    m_levelInterpType[0]          = MGRInterpolationType::jacobi;
+    m_levelRestrictType[0]        = MGRRestrictionType::injection;
+    m_levelCoarseGridMethod[0]    = MGRCoarseGridMethod::nonGalerkin;
+    m_levelGlobalSmootherType[0]  = MGRGlobalSmootherType::none;
 
     // Level 1
     m_levelFRelaxType[1]          = MGRFRelaxationType::gsElimWInverse;
-    m_levelFRelaxIters[1]        = 1;
-    m_levelInterpType[1]         = MGRInterpolationType::blockJacobi;
-    m_levelRestrictType[1]       = MGRRestrictionType::injection;
-    m_levelCoarseGridMethod[1]   = MGRCoarseGridMethod::galerkin;
-    m_levelGlobalSmootherType[0] = MGRGlobalSmootherType::none;
+    m_levelFRelaxIters[1]         = 1;
+    m_levelInterpType[1]          = MGRInterpolationType::blockJacobi;
+    m_levelRestrictType[1]        = MGRRestrictionType::injection;
+    m_levelCoarseGridMethod[1]    = MGRCoarseGridMethod::galerkin;
+    m_levelGlobalSmootherType[1]  = MGRGlobalSmootherType::none;
```

Comment:
what does the `m_levelGlobalSmootherType[0]` now get set to?

Reply:
none. It's a few lines above.

Reply:
This was a bug fix by the way